examples/yaml/control_wrfchem_aircraft.yaml

```diff
@@ -0,0 +1,141 @@
+# General Description:  
+# Any key that is specific for a plot type will begin with ts for timeseries, ty for taylor
+# Opt: Specifying the variable or variable group is optional
+# For now all plots except time series average over the analysis window. 
+# Seting axis values - If set_axis = True in data_proc section of each plot_grp the yaxis for the plot will be set based on the values specified in the obs section for each variable. If set_axis is set to False, then defaults will be used. 'vmin_plot' and 'vmax_plot' are needed for 'timeseries', 'spatial_overlay', and 'boxplot'. 'vdiff_plot' is needed for 'spatial_bias' plots and'ty_scale' is needed for 'taylor' plots. 'nlevels' or the number of levels used in the contour plot can also optionally be provided for spatial_overlay plot. If set_axis = True and the proper limits are not provided in the obs section, a warning will print, and the plot will be created using the default limits.
+analysis:
+  start_time: '2019-09-05-12:00:00' #UTC
+  end_time: '2019-09-06-00:00:00' #UTC
+  output_dir: /wrk/charkins/melodies_monet/aircraft/analysis #Opt if not specified plots stored in code directory.
+  debug: True
+  resample: '600S' #6 min so works on Hera as a test. Can comment this if submitting a job.
+model:
+  wrfchem_v4.2: # model label
+    # files: /wrk/d2/rschwantes/wrf/firex_mech/qzhu/run_CONUS_fv19_BEIS_1.0xISO_RACM_v4.2.2_racm_berk_vcp_noI_phot_soa/0905/*
+    files: /wrk/d2/rschwantes/wrf/firex_mech/qzhu/run_CONUS_fv19_BEIS_1.0xISO_RACM_v4.2.2_racm_berk_vcp_noI_phot_soa/0905/*
+    mod_type: 'wrfchem'
+    mod_kwargs: 
+      mech: 'racm_esrl_vcp'
+    radius_of_influence: 12000 #meters
+    mapping: #model species name : obs species name
+      firexaq:
+        no2: NO2_CL_RYERSON
+        'no': 'NO_CL_RYERSON'
+        #PM2_5_DRY: "PM2.5"
+        o3: "O3_CL_RYERSON"
+    variables:
+        'pres_pa_mid':
+            rename: pressure_model # name to convert this variable to, use 'pressure_model' for aircraft obs
+            unit_scale: 1 #Opt Scaling factor 
+            unit_scale_method: '*' #Opt Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        'temperature_k':
+            rename: temp_model # name to convert this variable to
+            unit_scale: 1 #Opt Scaling factor 
+            unit_scale_method: '*' #Opt Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+    projection: None
+    plot_kwargs: #Opt
+      color: 'dodgerblue'
+      marker: '^'
+      linestyle: ':' 
+obs:
+  firexaq: # obs label
+    filename: '/wrk/d2/rschwantes/obs/firex-aq/R1/10s_merge/firexaq-mrg10-dc8_merge_20190905_R1.ict'
+    obs_type: aircraft
+    variables: #Opt 
+      'O3_CL_RYERSON':
+        unit_scale: 1 #Opt Scaling factor 
+        unit_scale_method: '*' #Opt Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        nan_value: -777777 # Opt Set this value to NaN
```

Comment:
I've added in a way to replace the LLOD in the latest commit. 

To be consistent with the use of nan_value in the yaml file, I used LLOD_value=-888888 (the value as it is in the original dataset) and LLOD_setvalue=0.0 (the value to replace -888888 with). If there's a better name than LLOD_setvalue that you can think of I can change it. I figured having LLOD_value correspond to the value after replacing and nan_value correspond to the value prior to replacing would be confusing.

I also set this up so that LLOD_setvalue replaces LLOD_value **after** any unit conversion occurs. This is also noted in a comment in the control file.